packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -0,0 +1,303 @@
+import { IStyle, HighContrastSelector, ScreenWidthMaxSmall, getScreenSelector, getFocusStyle } from '../../Styling';
+import { IMessageBarStyleProps, IMessageBarStyles, MessageBarType } from './MessageBar.types';
+import { IsFocusVisibleClassName } from '@uifabric/utilities';
+
+export const getStyles = (props: IMessageBarStyleProps): IMessageBarStyles => {
+  const { theme, className, messageBarType, onDismiss, actions, truncated, isMultiline, expandSingleLine } = props;
+  const { semanticColors, palette, fonts } = theme;
+
+  const SmallScreenSelector = getScreenSelector(0, ScreenWidthMaxSmall);
+
+  const errorStyle = {
+    background: semanticColors.errorBackground,
+    selectors: {
+      '& .icon': {
+        color: semanticColors.errorText
```

Comment:
Thanks for converting these!
On top of errorBackground you should use warningText.

  /**
   * The default color of error text, used on bodyBackground.
   */
  errorText: string;
  /**
   * The color of text on errorBackground, warningBackground, blockingBackground, or successBackground.
   */
  warningText: string;

Reply:
The text color for all message bar types is covered by color: palette.neutralPrimary on the root element. The icon color for the error message bar should be errorText.

Reply:
I'd assume the icons should be the same color as the text though? I'm not sure where we stand with contrast requirements for those because they aren't required for understanding what's going on.

Reply:
The existing MessageBar uses different styles for the icons and the text. I'd assume the converted component should have the same behavior.
<img width="567" alt="error message bar" src="https://user-images.githubusercontent.com/6687333/41567740-a8c5f4ce-7316-11e8-86e7-c7b8c12f7990.png">
<img width="445" alt="screen shot 2018-06-18 at 4 43 27 pm" src="https://user-images.githubusercontent.com/6687333/41567781-de01db58-7316-11e8-93cc-6e1820546d7f.png">
<img width="511" alt="screen shot 2018-06-18 at 4 43 38 pm" src="https://user-images.githubusercontent.com/6687333/41567786-e2f15a30-7316-11e8-8f65-8286e316917d.png">
<img width="232" alt="screen shot 2018-06-18 at 4 44 07 pm" src="https://user-images.githubusercontent.com/6687333/41567792-e82ec76c-7316-11e8-96d7-fbf12d474899.png">